doc/nrf/protocols/wifi/index.rst

```diff
@@ -37,3 +37,4 @@ If you want to go through an online training course to familiarize yourself with
    debugging
    raw_tx_operation
    sap
+   sniffer_rx_operation
```

Comment:
Club this with `raw_tx_operation` (so that they appear one below the other on the page) and move both of them below `sap`